api/query/cache/monitor/monitor_test.go

```diff
@@ -64,6 +64,7 @@ func TestDoubleStart(t *testing.T) {
 		defer wg.Done()
 		err1 = mon.Start()
 	}()
+	time.Sleep(time.Microsecond * 100)
```

Comment:
Debugging left in? If not, can you add a comment?

Reply:
No; fixes a flaky test.